commit 64f8d4e872845f804a60f13e457e4df62cfbab61
Author: Gonçalo Almeida <goncalo_almeida_635@hotmail.com>
Date:   Mon Dec 11 16:12:43 2023 +0000

    Fixed Windows CI builds (#586)
    
    * Revert "Add gtest to Fetch content #550 (#551)"
    
    This reverts commit 70d59ed - breaking the windows builds
    
    * Fix CI for Windows builds
    
    ---------
    
    Co-authored-by: Gonçalo Almeida <Goncalo.Almeida@ctw.bmwgroup.com>

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ab399a6..82ab4b3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -22,10 +22,7 @@ if (NOT GTEST_ROOT)
     if (DEFINED ENV{GTEST_ROOT})
         set(GTEST_ROOT $ENV{GTEST_ROOT})
     else()
-        include(FetchContent)
-        FetchContent_Declare(googletest GIT_REPOSITORY https://github.com/google/googletest.git GIT_TAG v1.14.0) 
-        FetchContent_Populate(googletest)
-        set(GTEST_ROOT ${googletest_SOURCE_DIR})
+        set(GTEST_ROOT "n/a" CACHE STRING "Path to root folder of googletest. Must be set for building the tests.")
     endif()
 endif()
 
